app/job_creator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative 'database_manager'
+
+class JobCreator
+  def initialize(job_data)
+    @job_data = job_data
+  end
+
+  def create_job
+    existing_job = Job.find_by(url: @job_data[:url])
```

Comment:
Extract this method in private section and use it here

private 

def find_job ...